src/components/ProductCardList.jsx

```diff
@@ -0,0 +1,22 @@
+import { motion } from 'framer-motion';
+import ProductCard from './ProductCard';
+
+const ProductCardList = ({ products, title }) => {
+  return (
+    <div className='flex flex-col mt-10'>
+      <h1 className='text-left font-bold text-3xl mb-4 mt-5 text-primary '>{title}</h1>
+      <div className='carousel mt-4 overflow-hidden'>
+        {/* Make the 'motion.div' draggable along the horizontal axis and 'left: -500' means the element can be dragged up to 500 pixels to the left of its starting position. */}
+        <motion.div className='flex space-x-2' drag='x' dragConstraints={{ left: -500, right: 0 }}>
```

Comment:
We should also the grid layout in desktop view while maintaining the scrolling content in mobile. To achieve that, I think we can try

```javascript
      <div className='flex overflow-x-auto lg:overflow-x-visible lg:grid lg:grid-cols-[repeat(auto-fill,_minmax(240px,_1fr))] gap-8 p-4 no-scrollbar'>
{products.map((product, index) => (
            // key={index} provides a unique key for each ProductCard component
            <ProductCard key={index} product={product} />
          ))}
     </div>


```